raco/algebra.py

```diff
@@ -1525,25 +1537,35 @@ def __hash__(self):
         return ("%s-%s" % (self.opname(), self.relation_key)).__hash__()
 
     def shortStr(self):
-        return "%s(%s)" % (self.opname(), self.relation_key)
+        return "%s(%s, %s)" % (self.opname(),
+                               self.relation_key,
+                               "debroad" if self._debroadcast else "")
```

Comment:
Nitpick: I don't like this. An alternate I can think of is to have BroadcastScan(<<self. relation_key>>) and DeBroadcastScan(<<self. relation_key>>), but thats not pretty either.